engine/notifications.js

```diff
@@ -0,0 +1,283 @@
+import redis from '../engine/redis-helper.js';
+import webPush from 'web-push';
+
+const ttl = 2419200;
```

Comment:
This is already the default in the `web-push` library, so we can avoid setting it.